scripts/ci/install

```diff
@@ -11,15 +11,17 @@ fi
 wget -qO- "https://raw.githubusercontent.com/travis-ci/artifacts/master/install" |bash
 
 # install Miniconda3
-wget -nv "http://repo.continuum.io/miniconda/${MINICONDA_FILENAME}"
-bash ${MINICONDA_FILENAME} -b -f -p ${HOME}/miniconda
+if  [ "${NO_INSTALL_CONDA}" != "1" ]; then
+    wget -nv "http://repo.continuum.io/miniconda/${MINICONDA_FILENAME}"
+    bash ${MINICONDA_FILENAME} -b -f -p ${HOME}/miniconda
 
-# if emergency, temporary package pins are necessary, they can go here
-PINNED_PKGS=$(cat <<EOF
+    # if emergency, temporary package pins are necessary, they can go here
+    PINNED_PKGS=$(cat <<EOF
 pandas <0.23.0
```

Comment:
an `EOF` string looks really odd inside an indented block. I'd say let's move the definition of `PINNED_PKGS` outside above the conditional. There's no harm if it doesn't get used income cases

Reply:
I'd still prefer if `PINNED_PKGS` def is pulled out of the conditional so it isn't weirdly indented and not indented